molecule/rolling_update/converge.yml

```diff
@@ -13,7 +13,7 @@
     cartridge_scenario:
       - deliver_package
       - update_package
-    cartridge_package_path: ./packages/myapp-5.0.0-0-with-c-2.5.0.tar.gz
+    cartridge_package_path: ./packages/myapp-5.0.0-0-with-c-2.6.0.tar.gz
```

Comment:
I can be wrong, but the reason for testing special versions was the difference in the internal Cartridge API.

Reply:
2.4.0 doesn't work :)

More info here:
https://github.com/tarantool/sdk/issues/245

But it used only to update from the old version to the new version, so 2.5.0 + 2.6.0 is good too